scene/2d/navigation_agent_2d.cpp

```diff
@@ -482,7 +482,7 @@ void NavigationAgent2D::update_navigation() {
 			Dictionary details;
 
 			const Vector2 waypoint = navigation_path[navigation_path_index];
-			details[SNAME("location")] = waypoint;
+			details[SNAME("position")] = waypoint;
```

Comment:
The documentation for this signal in both the 3D and 2D node is still referring to the old name.